Dockerfile

```diff
@@ -20,4 +20,4 @@ RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates
 
 COPY --from=builder /build/ /
 
-CMD ["/feederd-linux","-debug","-conf=./data/config.json"]
\ No newline at end of file
+CMD ["/feederd-linux"]
```

Comment:
we assume to have the config.json to be mounted in `/` ?

Reply:
Yes: https://github.com/TDex-network/tdex-feeder/blob/2a66c3edb0d288fa46e18d7d1a3e932ec91a0fc4/README.md#L53
I don't see any advantage to give the possibility to change that